File: python_solutions/how-much-water.py
def how_much_water(water, load, clothes):
    # Good luck!
    formula = water * 1.1 ** (clothes - load)
    if clothes > load * 2:
        return 'Too much clothes'
    elif clothes < load:
        return 'Not enough clothes'
    else:
        return round(formula, 2)

File: javascript_solutions/how-much-water.js
function howMuchWater(water, load, clothes) {
	// Insert your brilliant code here
	let formula = water * 1.1 ** (clothes - load);
	if (clothes > load * 2) {
		return 'Too much clothes';
	} else if (clothes < load) {
		return 'Not enough clothes';
	} else {
		return parseFloat(formula.toFixed(2));
	}
}
